README.md

```diff
@@ -101,17 +101,17 @@ The Accessibility enum is used to determine when your secrets can be accessed. I
 
 ```swift
 let username = "Skroob"
-myValet.set(string: "12345", forKey: username)
+try? myValet.setString("12345", forKey: username)
```

Comment:
Thoughts on whether we should show the error being ignored? Both here and in the Objective-C code below. Previously we were discarding the return type, so the current sample-code maintains the prior behavior.

Reply:
Seems reasonable to keep the example simple. The presence of `try?` implies that there is an error case that could (should) be handled in real usage.